datafusion/tests/sql.rs

```diff
@@ -1694,6 +1694,17 @@ async fn equijoin() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn equijoin_unsuppored_condition() -> Result<()> {
```

Comment:
this test appears to be mis-named now. Perhaps something like

```suggestion
async fn equijoin_and_other_condition() -> Result<()> {
```